lib/sequenceserver/job.rb

```diff
@@ -109,10 +113,11 @@ def store(key, value)
       filename
     end
 
-    # Marks the job as done.
-    def done!
+    # Marks the job as done and save its exitstatus
+    def done!(status)
       @completed_at = Time.now
       @done = true
```

Comment:
Let's do away with setting `@done = true`. Instead update `Job#done?` to `!!exitstatus`.